package.json

```diff
@@ -1,115 +1,56 @@
-    "@types/ember-testing-helpers": "^0.0.4",
-    "@types/ember__test-helpers": "^2.6.1",
-    "@types/qunit": "^2.11.3",
-    "@types/rsvp": "^4.0.4",
-    "@typescript-eslint/eslint-plugin": "^5.11.0",
-    "@typescript-eslint/parser": "^5.11.0",
-    "broccoli-asset-rev": "^3.0.0",
-    "ember-animated-tools": "^0.5.0",
-    "ember-cli": "~4.1.1",
-    "ember-cli-dependency-checker": "^3.2.0",
-    "ember-cli-fastboot": "^3.2.0",
-    "ember-cli-fastboot-testing": "^0.6.0",
-    "ember-cli-inject-live-reload": "^2.1.0",
-    "ember-cli-sass": "^10.0.0",
-    "ember-cli-terser": "^4.0.2",
-    "ember-cli-typescript-blueprints": "^3.0.0",
-    "ember-composable-helpers": "^5.0.0",
-    "ember-concurrency": "^2.2.1",
-    "ember-data": "~4.1.0",
-    "ember-disable-prototype-extensions": "^1.1.3",
-    "ember-export-application-global": "^2.0.1",
-    "ember-load-initializers": "^2.1.2",
-    "ember-page-title": "^7.0.0",
-    "ember-qunit": "^5.1.5",
-    "ember-resolver": "^8.0.3",
-    "ember-source": "~4.1.0",
-    "ember-source-channel-url": "^3.0.0",
-    "ember-template-lint": "^4.0.0",
-    "ember-truth-helpers": "^3.0.0",
-    "ember-try": "^2.0.0",
-    "eslint": "^7.32.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-ember": "^10.5.8",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-qunit": "^7.2.0",
-    "faker": "5.5.3",
-    "loader.js": "^4.7.0",
-    "npm-run-all": "^4.1.5",
-    "prettier": "^2.5.1",
-    "qunit": "^2.17.2",
-    "qunit-dom": "^2.0.0",
-    "sass": "^1.17.3",
-    "typescript": "^4.5.4",
-    "webpack": "^5.68.0"
-  },
-  "engines": {
-    "node": "12.* || 14.* || >= 16"
+    "release-it": "^14.2.1",
+    "release-it-lerna-changelog": "^3.1.0",
+    "release-it-yarn-workspaces": "^2.0.0"
   },
-  "ember": {
-    "edition": "octane"
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org"
   },
-  "ember-addon": {
-    "configPath": "tests/dummy/config"
+  "release-it": {
+    "plugins": {
+      "release-it-lerna-changelog": {
+        "infile": "CHANGELOG.md",
+        "launchEditor": false
+      },
+      "release-it-yarn-workspaces": {
+        "workspaces": [
+          "addon"
+        ],
+        "additionalManifests": {
+          "dependencyUpdates": [
+            "test-app/package.json"
+          ]
+        }
+      }
+    },
+    "git": {
+      "tagName": "v${version}"
+    },
+    "github": {
+      "release": true,
+      "tokenRef": "GITHUB_AUTH"
+    },
+    "npm": false
   },
   "volta": {
     "node": "12.22.7",
     "yarn": "1.22.17"
+  },
+  "resolutions": {
+    "@ember/legacy-built-in-components": "github:emberjs/ember-legacy-built-in-components#a84513d197eeafbb01eae81e0f101b86ccde0ecf",
+    "@ember/render-modifiers": "github:SergeAstapov/ember-render-modifiers#ember-source-asterisk",
+    "ember-element-helper": "github:SergeAstapov/ember-element-helper#ember-source-asterisk",
```

Comment:
@ef4 FYI could not get peer deps working with pnpm so found out it's enough to [change `ember-source` in peerDependencies](https://github.com/SergeAstapov/ember-element-helper/commit/d2336cbcbd910e82dc5c8fc87897dc183ff8fa72) of [`@ember/render-modifiers`](https://github.com/SergeAstapov/ember-render-modifiers/commit/589344165122dda90b34b1e3ea3fdc37b40a115a) and [`ember-element-helper`](https://github.com/SergeAstapov/ember-element-helper/commit/d2336cbcbd910e82dc5c8fc87897dc183ff8fa72)

I'm not 100% confident this will always work as expected so will keep looking into pnpm.

meanwhile build is green and I can start actual v2 conversion